frontend/src/__tests__/cypress/cypress/e2e/modelServing/ModelServingGlobal.cy.ts

```diff
@@ -296,10 +306,30 @@ describe('Model Serving Global', () => {
     // add trailing space
     deleteModal.findInput().type(' ');
     deleteModal.findSubmitButton().should('be.disabled');
+
+    deleteModal.findInput().clear().type('Test Inference Service');
+    deleteModal.findSubmitButton().should('be.enabled');
+
+    deleteModal.findSubmitButton().click();
+
+    cy.wait('@deleteModel').then((interception) => {
+      //eslint-disable-next-line @typescript-eslint/no-unused-expressions
+      expect(interception.request).to.exist;
+    });
   });
```

Comment:
It's sufficient to know that the `DELETE` request was made here. Unless we are interested in asserting specific request data, you can remove the extra existence check.
```suggestion
    cy.wait('@deleteModel');
```

Reply:
You want me to remove `expect(interception.request).to.exist;` from other tests as well. we also have this check in `ModelMetrics.cy.ts` .

Reply:
If the wait targets a specific method and URL, then as soon as the wait passes, it implies there was a request. Therefore you can remove the other instances that do this exact same check.